src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -183,8 +188,8 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConfirmBidState
       })
 
       commitMutation(this.props.relay.environment, {
-        onCompleted: (_, errors) => (isEmpty(errors) ? this.createBidderPosition() : this.presentErrorResult(errors)),
-        onError: this.presentErrorResult.bind(this),
+        onCompleted: (_, errors) => (isEmpty(errors) ? this.createBidderPosition() : this.presentErrorModal()),
```

Comment:
Let's pass in `errors` to `presentErrorModal` and send it to Sentry so we'll know what errors are common and what errors are not:

```ts
onCompleted: (_, errors) => (isEmpty(errors) ? ... : this.presentErrorModal(errors)),
```

```ts
presentErrorModal(errors) {
  console.error("ConfirmBid.tsx", errors)

  ...
}
```

The generic `errorMessage ` looks good to me.

Reply:
Ahh yeah, I always end up disabling that while developing since it gets annoying-- will add back 👍